packages/grafana-ui/src/types/icon.ts

```diff
@@ -201,7 +201,7 @@ export const getAvailableIcons = () => availableIcons;
 
 type BrandIconNames = typeof avaibleBrandIcons;
 
-export type IconName = ReturnType<typeof getAvailableIcons>[number] | BrandIconNames[number];
+export type IconName = ReturnType<typeof getAvailableIcons>[number] | BrandIconNames[number] | string;
```

Comment:
I _think_ this causes editors to no longer show suggestions for icons, which seems like a huge miss to me.